tests/utils/v2Data/rand.js

```diff
@@ -0,0 +1,27 @@
+function range(n, step) {
```

Comment:
Wait... is this from another project? I swear I remember commenting that this can be a generator before...